website/server/controllers/api-v3/tasks.js

```diff
@@ -634,9 +634,18 @@ api.addTagToTask = {
     if (validationErrors) throw validationErrors;
 
     let taskId = req.params.taskId;
-    let task = await Tasks.Task.findByIdOrAlias(taskId, user._id, { userId: user._id });
+    let task = await Tasks.Task.findByIdOrAlias(taskId, user._id);
+
+    if (!task) {
```

Comment:
I'm not seeing wherein the group.syncTask function that the tags get synced. This should be exactly like challenges. Am I missing something?